frontend/src/app/shared/header/header.component.html

```diff
@@ -0,0 +1,22 @@
+<nav>
+  <img
+    src="./../../../assets/static/images/logo.png"
+    alt="website logo"
+    class="logo"
+  />
+  <ul class="left-nav">
+    <li><a routerLink="/home">Home</a></li>
+    <li><a routerLink="/products">Products</a></li>
+  </ul>
+  <ul class="right-nav">
+    <li><a routerLink="/login">Login</a></li>
+    <li><a routerLink="/register">Register</a></li>
+  </ul>
+  <ul class="cart-nav">
+    <li>
+      <a routerLink="/cart">
+        <img src="./../../../assets/static/images/cart.svg" />
+      </a>
```

Comment:
I guess there's a cart item count in design that we need to show?

Reply:
yes displayed